crates/runtime/src/traphandlers.rs

```diff
@@ -311,7 +381,7 @@ impl<T: Copy> Drop for ResetCell<'_, T> {
 // happen which requires us to read some contextual state to figure out what to
 // do with the trap. This `tls` module is used to persist that information from
 // the caller to the trap site.
-mod tls {
+pub(crate) mod tls {
```

Comment:
Is this still necessary? Ideally this'd stay private to this module to avoid having to reason about the rest of the crate

Reply:
The `backtrace` module has two calls to `tls::with(...)`. I guess we could do `pub(super::backtrace)` or whatever the syntax is (I think I've only ever used `pub(crate)` and `pub(super)` before). Or we could make the `backtrace` module a submodule of `traphandlers`.

Reply:
Ah ok, no need to do `pub(super::backtrace)`, but your other suggestion of moving `backtrace` as a submodule sounds reasonable to me.